workspaces/analytics/plugins/analytics-module-matomo/README.md

```diff
@@ -50,6 +50,7 @@ app:
       host: ${ANALYTICS_MATOMO_INSTANCE_URL}
       siteId: ${ANALYTICS_MATOMO_SITE_ID}
       identity: optional # (optional) to enable user tracking. Is disabled by default
+      disableEncryption: optional # (optional) to disable encryption of userId when user tracking is enabled. Encryption is enabled by default.
```

Comment:
as we are just hashing the user id, perhaps we can find a better naming for the new config. Some suggestions:
- `userIdFormat: "plain" | "hashed"`
- `sendPlainUserId: boolean`
- `userIdMode: "hashed" | "original"`

WDYT?